src/main/java/com/flab/marketflea/controller/ShopController.java

```diff
@@ -0,0 +1,53 @@
+package com.flab.marketflea.controller;
+
+
+import com.flab.marketflea.common.SessionService;
+import com.flab.marketflea.model.shop.ShopRequest;
+import com.flab.marketflea.service.shopservice.ShopService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+import static com.flab.marketflea.common.ResponseEntityConstants.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/shops")
+public class ShopController {
+
+
+    private final ShopService shopService;
+    private final SessionService sessionService;
+
+    @ResponseStatus(value = HttpStatus.CREATED)
+    @PostMapping("shops")
+    public void crateShop(@RequestBody ShopRequest shop) {
+        shopService.createShop(shop);
+    }
+
+    @GetMapping("/{shopId}/duplicate")
+    public ResponseEntity<Void> isIdDuplicated(@PathVariable long shopId) {
+        boolean isIdDuplicated = shopService.isShopExist(shopId);
+        if (isIdDuplicated) {
+            return CONFLICT;
+        } else {
+            return OK;
+        }
```

Comment:
굳이 else문을 쓸 필요는 없습니다

```suggestion
        if (isIdDuplicated) {
            return CONFLICT;
        }
        return OK;
```